### PR TITLE
[63] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by bint-saad**
[63] Style the Contact Us Page using CSS
[63] Style the Contact Us Page using CSS
con1
con3
con2

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
Kindly fix codefactor issue

**Comment by bint-saad** [comment]
con1
con3
con2

**Comment by bint-saad** [comment]
conflicts resolved..... kindly review

File: submissions/hadizaSaadu/css/contact-page.css [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+  color: #ffffff;
+  font-weight: bold;
+  float: right;
+}
+.logos {
+  width: 60px;
+  height: 40px;
+  padding-right: 10px;
+}
+/* .images {
```
Comment (adaugochi) [inline submissions/hadizaSaadu/css/contact-page.css:179]: remove it if not in use

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by bint-saad** [comment]
Removed

**Review by adaugochi — APPROVED** [review]

**Pull request merged**